agent/src/main/java/org/apache/airavata/mft/agent/MFTAgent.java

```diff
@@ -100,94 +100,92 @@ public void init() {
 
```

Comment:
Did you do auto formatting in this file? And what is the relevance of this change to ftp transport? If you are doing any change to agent or core, please raise them as separate PRs